usaspending_api/common/etl/primatives.py

```diff
@@ -47,7 +54,7 @@ def make_change_detector_conditional(columns: List[str], left_alias: str, right_
     return SQL(" or ").join(composed_conditionals)
 
 
-def make_column_list(columns: List[str], alias: Optional[str] = None) -> Composed:
+def make_column_list(columns: List[str], alias: str = None, overrides: ColumnOverrides = None) -> Composed:
```

Comment:
Shouldn't Optional[] have been left if the default values are None?
https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
I had a problem with Optional that caused me to yank it in a few places.  Sadly, I can't remember now what that problem was.  I'll add it back and see if I can jog my memory.